src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,681 @@
+                provider: fontProviders.google(),
+                name: "Roboto",
+            }]
+        }
+    })
+    ```
+
+3. Define a CSS variable for referencing the font family registered in your CSS.
+
+    This name can be anything you want, as long as it is a valid [CSS identifier](https://developer.mozilla.org/en-US/docs/Web/CSS/ident). You may wish to choose an identifier that resembles a CSS variable, but this is not required. 
+
+    ```js title="astro.config.mjs" ins={8}
+    import { defineConfig, fontProviders } from "astro/config"
+
+    export default defineConfig({
+        experimental: {
+            fonts: [{
+                provider: fontProviders.google(),
+                name: "Roboto",
+                cssVariable: "--font-roboto",
+            }]
+        }
+    })
+    ```
+
+</Steps>
+
+More configuration options, such as defining fallback font families, are available and some will depend on your provider. See the full [configuration reference](#font-configuration-reference) to learn more.
+
+## Applying fonts
+
+The `<Font />` component outputs style tags and (optionally) preload links for a given font family. 
+
+It must be imported and added to your page `<head>` and `family` is required. Pass the `cssVariable` value from your configuration.
+
+This is commonly done in a component such as `Head.astro` that is used in a common site layout:
+
+```astro title="src/components/Head.astro" ins={2, 5}
+---
+import { Font } from 'astro:assets'
+---
+
+<Font family="--font-roboto" preload />
+```
+
+<ReadMore>See the full [`<Font>` component reference](#font--component-reference) for more information.</ReadMore>
+
+Since the `<Font />` component generates CSS with font declarations, you can reference the font family using the `cssVariable` defined in your Astro configuration:
+
+```css
+<style>
+body {
+    font-family: var(--font-roboto);
+}
+</style>
+```
+
+If you use Tailwind CSS, you can use the following configuration:
+
+<Tabs>
+
+<TabItem label="Tailwind CSS 4.0">
+
+```css title="src/styles/global.css" ins={4}
+@import 'tailwindcss';
+
+@theme {
+    --font-sans: var(--font-roboto);
+}
+```
+
+</TabItem>
+
+<TabItem label="Tailwind CSS 3.0">
+
+```js title="tailwind.config.mjs" ins={6-8}
+/** @type {import("tailwindcss").Config} */
+export default {
+  content: ["./src/**/*.{astro,html,js,jsx,md,mdx,svelte,ts,tsx,vue}"],
+  theme: {
+    extend: {},
+    fontFamily: {
+        sans: ["var(--font-roboto)"]
+    }
+  },
+  plugins: [],
+};
+```
+
+</TabItem>
+
+</Tabs>
+
+## `<Font />` component reference
+
+This component outputs style tags and preload links for a given font family.
+
+```astro
+---
+import { Font } from 'astro:assets'
+---
+```
+
+### family
+
+<p>
+
+**Type:** `string`
+</p>
+
+The [`cssVariable`](#cssvariable) registered in your Astro configuration:
+
+```astro title="src/components/Head.astro" "family"
+---
+import { Font } from 'astro:assets'
+---
+
+<Font family="--font-roboto" />
```

Comment:
Rename prop to `cssVariable`:
- update all occurrences
- update rfc
- code pr